lib/active_remote/cached.rb

```diff
@@ -71,74 +76,126 @@ def cached_search(argument_hash, options = {})
         end
       end
 
+      def method_missing(m, *args, &block)
+        method_name = _method_missing_name(m)
+
+        if method_name.nil? || !cached_methods.include?(method_name.to_s)
+          super(m, *args, &block)
+        else
+          new_args = _args_in_sorted_order(m, args)
+          __send__(method_name, *new_args, &block)
+          #super(m, *args, &block)  Institution.cached_find_by_institution_type_and_guid(:institution_type => 1, :guid => "asdf")
```

Comment:
I'm assuming this commented out line should be removed?

Reply:
yep, thanks for catching that.